templates/user-data.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+. /etc/environment
+USERDATA_FILE={{.MachineType}}
+
+/usr/bin/rkt run \
+    --net=host \
+    --volume=dns,kind=host,source=/etc/resolv.conf,readOnly=true --mount volume=dns,target=/etc/resolv.conf  \
+    --volume=awsenv,kind=host,source=/var/run/coreos,readOnly=false --mount volume=awsenv,target=/var/run/coreos \
+    --trust-keys-from-https \
+    quay.io/coreos/awscli:latest -- aws s3 --region {{.Region}} cp s3://{{.S3Bucket}}/{{.ClusterID}}/cloudconfig/$USERDATA_FILE /var/run/coreos/temp.txt
```

Comment:
Another issue I see here is about the paths used to copy the cloud config. The S3 path contains `cloudconfig` which is also used on the go code. When somebody changes the path it is quite likely to forget to change the path in here. So I would like to inject all the paths completely to have one place where we are able to change the locations.